### PR TITLE
Fixed Kotlin errors

### PR DESCRIPTION
**Pull request by hberndt**
After generating a Kotlin microservice the compile failed with some Kotlin issues.

In Kotlin exists no ternary operator, arrays are created by arrayOf and some imports were missing.

### CODE REVIEW

**Comment by sendilkumarn** [comment]
@hberndt thanks a lot for your contribution and congrats on your first 🎉

**Comment by hberndt** [comment]
@sendilkumarn thank you. I will go on.

File: generators/server/templates/src/main/kotlin/package/config/AsyncConfiguration.kt.ejs [hunk 1]
```diff
@@ -38,7 +38,7 @@ class AsyncConfiguration(private val jHipsterProperties: JHipsterProperties): As
 
     private val log = LoggerFactory.getLogger(this.javaClass)
 
-    @Bean(name = ["taskExecutor"])
+    @Bean(name = arrayOf("taskExecutor")])
```
[the remark this answers is not in the document]
Comment (hberndt) [reply generators/server/templates/src/main/kotlin/package/config/AsyncConfiguration.kt.ejs]: Amazing. My Kotlin Compiler shows errors with this syntax. [] can't be used within annotations in my environment and I think I am not the only one. I don't use IntelliJ

[the remark this answers is not in the document]
Comment (sendilkumarn) [reply generators/server/templates/src/main/kotlin/package/config/AsyncConfiguration.kt.ejs]: interesting. But this is just a warning. So it is fine to have it like this then

**Pull request merged**